tests/unit/controllers/survey/2016-test.js

```diff
@@ -1,14 +1,12 @@
-import { setupTest } from 'ember-qunit';
 import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
```

Comment:
Please undo all changes that are not necessary to make linting and testing pass. The goal is to keep your pull request small so that it's easy to review.

- Line 4, before the change, can change from `ember-community-survey-2016` to `survey/2016`.
- Line 9, before the change, can change from `'controller:ember-community-survey-2016'` to `'controller:survey/2016'`.

This ask applies to all unit tests from 2016 to 2021.